tests/e2e/tests/devfiles/PHPSimple.spec.ts

```diff
@@ -0,0 +1,75 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import 'reflect-metadata';
+import * as projectAndFileTests from '../../testsLibrary/ProjectAndFileTests';
+import * as workspaceHandling from '../../testsLibrary/WorksapceHandlingTests';
+import * as commonLsTests from '../../testsLibrary/LsTests';
+import * as codeExecutionTests from '../../testsLibrary/CodeExecutionTests';
+import { e2eContainer } from '../../inversify.config';
+import { WorkspaceNameHandler, Editor, CLASSES } from '../..';
+import { Logger } from '../../utils/Logger';
+
+const editor: Editor = e2eContainer.get(CLASSES.Editor);
+
+const workspaceSampleName: string = 'php-web-simple';
+const fileFolderPath: string = `${workspaceSampleName}`;
+const tabTitle: string = 'index.php';
+// const codeNavigationClassName: string = 'RouterImpl.class';
+const depTaskName: string = 'Configure Apache Web Server DocumentRoot';
+const buildTaskName: string = 'Start Apache Web Server';
+const buildTaskLinkExpectedText: string = 'A process is now listening on port 8080.';
+const stack: string = 'PHP Simple';
+
+suite(`${stack} test`, async () => {
+    suite (`Create ${stack} workspace`, async () => {
+        workspaceHandling.createAndOpenWorkspace(stack);
+        projectAndFileTests.waitWorkspaceReadinessNoSubfolder(workspaceSampleName);
+    });
+
+    suite('Language server validation', async () => {
```

Comment:
I would suggest to follow the same structure of the test that is in other tests - open the file first, then execute commands (so LS has enough time to start) and then execute LS tests.

Reply:
LS is initialized practically instantly, but I'm all for keeping the tests consistent. thank you for the notice.

Reply:
fixed.